packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -564,6 +577,21 @@ export class FluidDataStoreRuntime extends EventEmitter implements IFluidDataSto
                 return new TreeTreeEntry(key, snapshot);
             }));
 
+        // NOTE: Search blob concept (just testing one contract for now):
+        const searchContract = this.extraSnapshotContracts.get("search");
```

Comment:
I'm not sure what's the purpose of extraSnapshotContracts to have many entries if we are using just one.
I'd rather change it and say that contract extraSnapshotContracts is just a single function, that returns ITreeEntry[] that gets added to entries.

Reply:
I can do this. My thought was that since the point is to abstract search as just a "contract" between a particular DataObject and storage then in some cases you could have more than just one of these "extra" contracts.